AquariumShop.php

```diff
@@ -0,0 +1,84 @@
+<?php
+declare(strict_types=1);
+
+class FishType {
+    private $name;
+
+    private $character;
+
+    private $phMin;
+
+    private $phMax;
+
+    public function __construct(string $name, string $character, float $phMin, float $phMax) {
+        $this->name = $name;
+        $this->character = $character;
+        $this->phMin = $phMin;
+        $this->phMax = $phMax;
+    }
+
+    public function canLiveTogether(FishType $fishType) {
```

Comment:
please return boolean type, instead of printing output.
We should not mix presentation and business logic.